src/services/ui/src/features/package-actions/lib/modules/withdraw-package/index.tsx

```diff
@@ -73,7 +73,7 @@ export const defaultWithdrawPackageContent: FormContentHydrator = (
     header: "Withdraw package?",
     body: `You are about to withdraw ${document.authority} ${document.id}. Completing this action will conclude the review of this ${document.authority} package. If you are not sure this is the correct action to select, contact your CMS point of contact for assistance`,
     acceptButtonText: "Yes, withdraw package",
-    cancelButtonText: "Return to form",
+    cancelButtonText: "Cancel",
```

Comment:
I'm out of the loop on this. And it may have been there before. We are expected to change this text to cancel, correct?

Reply:
Indeed